features/leeftijd_bepaling.feature

```diff
@@ -35,11 +35,27 @@ Scenario: Alleen jaar van geboorte datum is bekend
 	Dan is attribuut leeftijd niet aanwezig
 
 Scenario: Persoon is overleden
-	Gegeven een ingeschreven persoon met geboortedatum 26 mei 1953
-	En de ingeschreven persoon is overleden
-	Als de ingeschreven persoon op <raadpleeg datum> wordt geraadpleegd
+	Gegeven de persoon met burgerservicenummer "999993197" is overleden op 28-04-2019
+	Als de ingeschreven persoon met burgerservicenummer 999994268 wordt geraadpleegd met fields=leeftijd
 	Dan is attribuut leeftijd niet aanwezig
 
+Scenario: leeftijd wordt wel geleverd bij een overleden kind (omdat alleen gegevens van de persoonslijst van de gevraagde persoon worden gebruikt)
+	Gegeven de persoon met burgerservicenummer "999991280" heeft een kind met burgerservicenummer "999993197" en voornamen "Henk-Pieter"
+	En de persoon met burgerservicenummer "999993197" is overleden op 28-04-2019
```

Comment:
Staat dit ook in categorie 09 van de persoon met bsn 999991280? Ik weet niet of ik dit uit dit gegeven kan concluderen

Reply:
@MelvLee of een kind is overleden staat niet in category 09. Dat is precies waar dit scenario voor is bedoeld, omdat overleden niet bekend is wordt leeftijd er niet uit gefilterd.